projects/shared/src/lib/service/user.service.ts

```diff
@@ -95,7 +95,7 @@ export class UserService {
       .pipe(map(data => {
         return data.metadata ? data.metadata : undefined;
       })).subscribe(user => {
-        if (user) {
+        if (user && user.libraries) {
```

Comment:
This conditions is a little strange to read as an human. Maybe better to test roles of user (librarian or system_librarian) or better create an util method `isStaffMember` ? But the result is the same at this time.

Reply:
I have changed the functionality